src/ui/AccountDropdown/index.svelte

```diff
@@ -38,41 +39,58 @@
         <a
           class="btn-ghost"
           href="https://app.santiment.net/alerts?tab=1"
-          on:click={window.__onLinkClick}>My alerts</a
-        >
+          on:click={window.__onLinkClick}>My alerts</a>
         <a class="btn-ghost" href="https://app.santiment.net/assets" on:click={window.__onLinkClick}
-          >My watchlists</a
-        >
+          >My watchlists</a>
         <a
           class="btn-ghost"
           href="https://insights.santiment.net/my"
-          on:click={window.__onLinkClick}>My insights</a
-        >
+          on:click={window.__onLinkClick}>My insights</a>
         <a
           href="https://insights.santiment.net/new"
           class="write btn-1 btn--s"
-          on:click={window.__onLinkClick}>Write insight</a
-        >
+          on:click={window.__onLinkClick}>Write insight</a>
       </section>
       <hr />
     {/if}
 
     <section>
+      {#if !currentUser}
+        <a href="/login" class="loginbtn" on:click={window.__onLinkClick}>
+          <Svg id="user" w="16" class="mrg-s mrg--r" />
+          <span>Log in</span>
+        </a>
+      {/if}
+
+      {#if appVersionState !== undefined && appVersionState >= 0}
+        <div class="appversion">
+          {#if appVersionState === 0}
+            <div class="latest">
+              You have the latest version!
+              <div class="caption c-waterloo">{versionLabel}</div>
+            </div>
+          {/if}
+          {#if appVersionState === 1}
+            <button class="latest btn-ghost fluid" on:click={() => window.location.reload()}>
+              Update available. Restart now
+            </button>
+          {/if}
+        </div>
+      {/if}
```

Comment:
There are only 2 options right now: 1) user has the latest version; 2) Update available.
Therefore only boolean variable is needed, e.g. `isAppUpdateAvailable`
```svelte
{#if isAppUpdateAvailable}
  ....
{:else}
  ....
{/if}
```